packages/farcaster-js-hub-rest/src/openapi/spec.yaml

```diff
@@ -796,6 +796,34 @@ paths:
         default:
           $ref: '#/components/responses/ErrorResponse'
       x-codegen-request-body-name: body
+  /v1/validateMessage:
```

Comment:
They merged the PR accepting this spec into the hubble-monorepo [here](https://github.com/farcasterxyz/hub-monorepo/blob/main/packages/hub-nodejs/spec.yaml)

Probably a good idea to duplicate this diff into a PR in their repo.

Ultimately we'll want to make their copy of spec.yaml the source of truth and remove our copy

Reply:
Agreed. I made a PR for the spec changes here - https://github.com/farcasterxyz/hub-monorepo/pull/1634